src/en/2017-04-er/03/04.md

```diff
@@ -1,7 +1,16 @@
 ---
-title:  Daily Lesson
+title:  Improvement? (Romans 1:22-32)
```

Comment:
@PrJared In this case if there is a `colon` in the title you should put the title in quotes for ex `"Improvement? (Romans 1:22-32)` But even better is to move the verse in the body so it will become clickable

Reply:
Ok, so I should move (Romans 1:22-32) down onto a new line?

How can I access and change the files after I've made a pull request?

Reply:
Are you unable to access the files after making a pull request? You should be able to.

Reply:
where do I do this?

Reply:
You can't open the files in the way that you usually do? I'm not quite sure of where you're stuck... If you could post a screenshot, that would help 🍟

Reply:
hang on, I found it!! Github is so confusing!  So how do I cancel this pull request, so I can make the changes and make a new pull request?

Reply:
You shouldn't have to make a new pull request... In fact, it's best practice not to do so. Some developers will ding you on the head if you do 😣

Reply:
yep, I fixed it.